src/test/java/org/jenkinsci/plugins/kubernetes/credentials/OpenShiftBearerTokenCredentialTest.java

```diff
@@ -35,7 +36,8 @@
 
     @Before
     public void prepareFakeOAuthServer() throws Exception {
-        server = new Server(0);
+        InetSocketAddress addr = new InetSocketAddress("localhost", 0);
```

Comment:
Just to understand, what does this change ?

Reply:
It change nothing in normal situation, I make the change because I was connected via VPN and maybe also due to proxy settings I was experiencing issues reaching the assigned IP (192.* ) . If needed , I can revert this change.